requirements/extra-audiobackend.txt

```diff
@@ -1,3 +1,3 @@
 pychromecast==3.2.2
 python-vlc>=1.1.2
-#git+https://github.com/JarbasAl/py_mplayer
\ No newline at end of file
+# TODO add ovos ocp + ovos vlc
```

Comment:
this TODO is for a future PR, note that vlc and simple audio backends have not been removed by this PR